src/assets/icons/chevron-down.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="10" height="6" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We already have an `angle-bracket-down.svg` icon, which is just a synonym for `chevron-down`. They should be identical. Please use the other und remove this. I tracked this issue here #261

Reply:
@AndyOGo The Problem is they are not identical, yours has rounded edges mine hasn't. 😄

Reply:
@TheDadi Ohh, though they are the same.
But why aren't we using the same arrows then? What is @anja225 saying?

Reply:
@TheDadi 
@LucaMele 
@anja225 
I just search through the material icon and found icons for **expand more/less**:
- https://material.io/icons/#ic_expand_more
- https://material.io/icons/#ic_expand_less

Reply:
if they are the same apart of round edges, better to have only 1 or not? @TheDadi can we not use the one already in the plib?